monai/handlers/classification_saver.py

```diff
@@ -134,6 +138,6 @@ def _finalize(self, engine: Engine) -> None:
 
         # save to CSV file only in the expected rank
         if idist.get_rank() == self.save_rank:
-            saver = CSVSaver(self.output_dir, self.filename, self.overwrite)
+            saver = CSVSaver(self.output_dir, self.filename, self.overwrite) if self.saver is None else self.saver
```

Comment:
The following would be slightly shorter
```suggestion
            saver = self.saver or CSVSaver(self.output_dir, self.filename, self.overwrite)
```

Reply:
sure, updated in the latest commit.
Thanks.